tomcat-conf/Catalina/localhost/prweb.xml

```diff
@@ -32,5 +32,13 @@
   <Environment name="prconfig/session/ha/enabled" value="true" type="java.lang.String"/>
   <Environment name="prconfig/dsm/services/stream/pyBaseLogPath" value="/opt/pega/kafkadata" type="java.lang.String" />
   <Environment name="prconfig/cloud/isNodeAgnosticAdminStudio" value="true" type="java.lang.String"/>
+
+  {{ if isTrue .Env.CLIENT_MODE }}
+  <Environment name="prconfig/cluster/clientserver/clientmode" value="{{ .Env.CLIENT_MODE }}" type="java.lang.String"/>
+  <Environment name="prconfig/Pega-RULES/onPegaK8s" value="{{ .Env.ON_PEGA_K8S }}" type="java.lang.String"/>
+  <Environment name="prconfig/identification/cluster/name" value="{{ .Env.CLUSTER_NAME }}" type="java.lang.String"/>
+  <Environment name="prconfig/cluster/clientserver/server/namespace" value="{{ .Env.NAMESPACE }}" type="java.lang.String"/>
```

Comment:
Instead of server namespace and service name, if we just used URL, wouldn't it cover both?

Reply:
Will make this change.

Reply:
A single URL won't change.  And if the DNS lookup changes, the deployment config would need to anyways.  By specifying a namespace, it is tied to the implementation of Kubernetes rather than just container orchestration.